jingo/__init__.py

```diff
@@ -49,7 +50,19 @@ def get_env():
     # TEMPLATE_DIRS and packages in INSTALLED_APPS.
     x = ((jinja2.FileSystemLoader, settings.TEMPLATE_DIRS),
          (jinja2.PackageLoader, settings.INSTALLED_APPS))
-    loaders = [loader(p) for loader, places in x for p in places]
+    loaders = []
```

Comment:
@jezdez any chance you have a minute to apply that change to this patch? i'm running into it with jingo + adminplus and i'm not really sure I understand that comment